verify.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, configfile, auth):
             elif opts[line] == '-d':
                 self.desc = opts[line + 1]
             elif opts[line] == '-b':
-                self.bin = opts[line + 1]
+                self.bin = self.desc
```

Comment:
It looks like they come in the order they were specified at the command line. Here's the one produced just now with the latest import/export tool:

```
-b
-r
http://localhost:8080/rest
-m
export
-d
/Users/westgard/Desktop/exporttest
```

I think the following will work, but since I'm github impaired, @ruebot, maybe you can update the PR?

```
elif opts[line] == '-d':
    self.desc = opts[line + 1]
    self.bin = self.desc
elif opts[line] == '-b':
    pass
```